components/Slider/src/ColorThemer/MDCSliderColorThemer.m

```diff
@@ -19,9 +19,45 @@
 
 static const CGFloat kSliderThemerLightAlpha = 0.26f;
 static const CGFloat kSliderThemerDarkAlpha = 0.3f;
+static const CGFloat kSliderM2BaselineDisabledFillAlpha = 0.32;
+static const CGFloat kSliderM2BaselineEnabledBackgroundAlpha = 0.24;
+static const CGFloat kSliderM2BaselineDisabledBackgroundAlpha = 0.12;
+static const CGFloat kSliderM2BaselineEnabledTicksAlpha = 0.54;
+static const CGFloat kSliderM2BaselineDisabledTicksAlpha = 0.12;
```

Comment:
Remove "SliderM2Baseline" prefix on all of these constants. We'll eventually remove the kSliderThemerLightAlpha and kSliderThemerDarkAlpha as well.